video/cloud-client/analyze/video_detect_person_beta.py

```diff
@@ -0,0 +1,99 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START video_detect_person_beta]
+import io
+from google.cloud import videointelligence_v1p3beta1 as videointelligence
+
+
+def detect_person(local_file_path="path/to/your/video-file.mp4"):
+    """Detects people in a video from a local file."""
+
+    client = videointelligence.VideoIntelligenceServiceClient()
+
+    with io.open(local_file_path, "rb") as f:
+        input_content = f.read()
+
+    # Configure the request
+    config = videointelligence.types.PersonDetectionConfig(
+        include_bounding_boxes=True,
+        include_attributes=True,
+        include_pose_landmarks=True,
+    )
+    context = videointelligence.types.VideoContext(
+        person_detection_config=config
+    )
+
+    # Start the asynchronous request
+    operation = client.annotate_video(
+        input_content=input_content,
+        features=[videointelligence.enums.Feature.PERSON_DETECTION],
+        video_context=context,
+    )
+
+    print("\nProcessing video for person detection annotations.")
+    result = operation.result(timeout=300)
+
+    print("\nFinished processing.\n")
+
+    # Retrieve the first result, because a single video was processed.
+    annotation_result = result.annotation_results[0]
+
+    for annotation in annotation_result.person_detection_annotations:
+        print("Person detected:")
+        for track in annotation.tracks:
+            print(
+                "Segment: {}s to {}s".format(
+                    track.segment.start_time_offset.seconds
+                    + track.segment.start_time_offset.nanos / 1e9,
+                    track.segment.end_time_offset.seconds
+                    + track.segment.end_time_offset.nanos / 1e9,
+                )
+            )
+
+            # Each segment includes timestamped objects that include
+            # characteristic - -e.g.clothes, posture of the person detected.
+            # Grab the first timestamped object
+            timestamped_object = track.timestamped_objects[0]
+            box = timestamped_object.normalized_bounding_box
+            print("\tBounding box:")
+            print("\t\tleft  : {}".format(box.left))
+            print("\t\ttop   : {}".format(box.top))
+            print("\t\tright : {}".format(box.right))
+            print("\t\tbottom: {}".format(box.bottom))
+
+            # Attributes include unique pieces of clothing,
+            # poses, or hair color.
+            print("\tAttributes:")
+            for attribute in timestamped_object.attributes:
+                print(
+                    "\t\t{}:{} {}".format(
+                        attribute.name, attribute.value, attribute.confidence
+                    )
+                )
+
+            # Landmarks in person detection include body parts
```

Comment:
maybe list a few landmarks in the comment, like left_shoulder, right_knee etc.

Reply:
Fixed in most recent commit